6.2.18 December release is out (#2265)
RedisLabs/redislabs-docs: content/rs/release-notes/rs-6-2-18.md

- Added support to the UI to work with custom REST API port (RS84428)
- Added info level print to redis_mgr for troubleshooting (RS85385)

## Enhancements added in 6.2.18-65 (December release)

> - When setting up an alerts email server in the UI: added a “To” field to the “test email server” function (RS86119)

Reviewer: ```suggestion
- When setting up an alert email server in the UI: added a “To” field to the “test email server” function (RS86119)
```

> - Added a new flag to skip cluster resources validation when performing import (RS88086)

Reviewer: ```suggestion
- Added a new flag to skip cluster resource validation when performing import (RS88086)
```

> - Enhanced error messages information for connection handling 

Reviewer: ```suggestion
- Enhanced error message information for connection handling 
```

> - Added logging information for planner’s error “insufficient_resources”

Reviewer: ```suggestion
- Added logging details to “insufficient_resources” planner error 
```

- RS77339 - Fixes the “FROM” email addresses of email alerts
- RS86005 - Fixes  'bdbs/<int:uid>/actions/recover' API to return the bdb status ans not the recovery_plan

## Resolved issues in 6.2.18-65 (December release)

> - RS87191 - Fixes syncer stop due to crosslot violation error in Redis on Flash

Reviewer: ```suggestion
- RS87191 - Fixes syncer stop due to cross slot violation error in Redis on Flash
```
